src/BrowserKitDriver.php

```diff
@@ -413,7 +413,24 @@ public function getValue($xpath)
      */
     public function setValue($xpath, $value)
     {
-        $this->getFormField($xpath)->setValue($value);
+        $formField = $this->getFormField($xpath);
+
+        try {
```

Comment:
For options, that have only title, but no value it's not possible for BrowserKit to select them, because BrowserKit only does value-based option selection.

This is a blocker for implementing select by option title functionality in this driver.